.github/workflows/vercel-deploy.yml

```
@@ -0,0 +1,38 @@
+name: Deploy to Vercel
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)